osu.Game/Rulesets/UI/DrawableRuleset.cs

```diff
@@ -98,6 +98,14 @@ internal override bool FrameStablePlayback
 
         private DrawableRulesetDependencies dependencies;
 
+        /// <summary>
```

Comment:
Can split these changes out if required.

Reply:
I think it's fine for a change as simple as this.